agent/src/util/config_action.rs

```diff
@@ -90,36 +90,35 @@ async fn watch_for_config_changes(
     new_discovery_handler_sender: broadcast::Sender<String>,
 ) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
     trace!("watch_for_config_changes - start");
-    let akri_config_type = RawApi::customResource(API_CONFIGURATIONS)
-        .group(API_NAMESPACE)
-        .version(API_VERSION);
-    let informer = Informer::raw(kube_interface.get_kube_client(), akri_config_type)
-        .init()
-        .await?;
-    loop {
-        let mut configs = informer.poll().await?.boxed();
-
-        // Currently, this does not handle None except to break the
-        // while.
-        while let Some(event) = configs.next().await {
-            let new_discovery_handler_sender = new_discovery_handler_sender.clone();
-            handle_config(
-                kube_interface,
-                event?,
-                config_map.clone(),
-                discovery_handler_map.clone(),
-                new_discovery_handler_sender,
-            )
-            .await?
-        }
+    let resource = Api::<Configuration>::all(kube_interface.get_kube_client());
+    let mut stream = resource
+        .watch(
+            &ListParams::default(),
+            akri_shared::akri::API_VERSION_NUMBER,
```

Comment:
curiousity: why move `akri_shared::akri::API_VERSION_NUMBER` out of the use block above?

Reply:
No particular reason. I created `API_VERSION_NUMBER` in this PR and just thought to import it in line since it is only used once here